ppmpy/ppm.py

```diff
@@ -12148,6 +12152,91 @@ def sphericalHarmonics_lmax(self, radius):
 
         return lmax, N, npoints
 
+    def get_power_spectrum(self, varloc, dump_start, dump_stop, dump_step=1, radius=None, mass=None):
+        '''
+        Calculates the power spectrum of any moms variable 
+
+        Power spectrum is calculated for variable 'varloc' at radius 'radius' for
+        dumps arange(dump_start, dump_stop+1, dump_step). The spectrum is averaged
+        over the ranfe of dumps given as inputs.
+        The power spectrum is expressed as power per spherical harmonic mode 'ell'.
+        
+        Note that power spectrums of velocities will be in 10^-12 m^2/s^2/ell
```

Comment:
are you sure about these units? I made a plot and for a M25 core convection I get low-ell power of 1e-6

the sum power over ell should give back the rad velocity, no? that is order 800m/s